docs/CONTRIBUTING.md

```diff
@@ -158,3 +158,12 @@ will run the tests for the specified rust version and specified image
 If you're member of **paritytech** org - you can use command-bot to run various of common commands in CI:
 
 Start with comment in PR: `bot help` to see the list of available commands.
+
+
+## Deprecating code
+
+Polkadot SDK does not stick to [SemVer](https://semver.org/), thus breaking changes can happen between "minor version"
+updates.
```

Comment:
We recently had a chat about this with @bkchr 

At rigor, we currently stick to SemVer, since we:
- always bump major versions on release of crates with version >= 1.0.0
- we always bump minor versions on release of crates with version < 1.0.0

Before v1.0.0 you can break stuff and just bump minor version https://semver.org/#spec-item-4

The only thing to improve IMO is that we bump versions even if we don't increment the `Cargo.toml` file of the crate. So probably it would be nice to add to the guidelines to actually bump major/minor versions in `Cargo.toml` inline with the `SemVer` rules